### ISSUE TITLE
Utilizing Salt and UITK In combination prevents styling from applying correctly

### ISSUE THREAD

**Issue by adamrideout** [jpmorganchase/salt-ds#2671]
### Package name(s)

Core (@salt-ds/core)

### Package version(s)

    "@salt-ds/core": "^1.10.0",

### Description

We're in the process of migrating our application from using UITK to using Salt. Since not all components are available yet, we have a mixture of the two. The problem is, the component styling from each is interfering with each other. For example, I'm trying to add a group of salt RadioButtons, but they end up completely misaligned. When I remove the `import "@jpmuitk/style/css/jpmuitk.css";` from our app.tsx, the styling returns to normal. However, without it the styling for all of our UITK components becomes completely messed up. What is the recommended process for adding salt styling and UITK styling without them interfering with each other? Thanks

### Steps to reproduce

Created a react application
Install UITK
Install SALT
Add 
```
import "@jpmuitk/style/css/jpmuitk.css";

import "@salt-ds/theme/index.css";
```
to your application.
Add salt radio buttons.

### Expected behavior

I expected the styling of the radio buttons to be correct, with the buttons aligned with the label

### Operating system

- [ ] macOS
- [X] Windows
- [ ] Linux
- [ ] iOS
- [ ] Android

### Browser

- [X] Chrome
- [ ] Safari
- [ ] Firefox
- [ ] Edge

### Are you a JPMorgan Chase & Co. employee?

- [X] I am an employee of JPMorgan Chase & Co.

___

Affected Components

- [x] Checkbox
- [x] Radio Button
- [ ] Banner

**Comment by james-nash** [comment]
Bug ranking score: **52** (using rubric v3)

Priority: **high**

```
========== BEGIN BUG RUBRIC SCORE DATA ===========
eyJ2YWx1ZXMiOnsiYWZmZWN0ZWRQZW9wbGVPcHRpb25zIjoxOS
wiYWZmZWN0ZWRDb21wb25lbnRzT3B0aW9ucyI6MTIsImZyZXF1
ZW5jeU9wdGlvbnMiOjE5LCJpbXBhY3RPcHRpb25zIjoyMCwiYW
ZmZWN0ZWREc09wdGlvbnMiOjE3LCJ1cmdlbmN5T3B0aW9ucyI6
MTksInhGYWN0b3JPcHRpb25zIjoxOH0sInJ1YnJpY1ZlcnNpb2
4iOjN9
=========== END BUG RUBRIC SCORE DATA ============
```

**Comment by origami-z** [comment]
The problem comes from the `line-height` in `jpmuitk.css`

```
body {
    font-weight: 400;
    font-family: jpmuitk-sans;
    font-size: 14px;
    line-height: 1.85714286em;
```

**Comment by james-nash** [comment]
Given that all Salt components have class names beginning with `salt...`, would something like this help maybe:

```css
[class^=salt] {
  line-height: initial;
}
```

**Comment by origami-z** [comment]
> Given that all Salt components have class names beginning with `salt...`, would something like this help maybe:
> 
> ```css
> [class^=salt] {
>   line-height: initial;
> }
> ```

This could be the workaround we provide. We still want all components are self contained and referring Salt line height setting, so can be tweaked via CSS variable

**Comment by adamrideout** [comment]
Just wanted to say thank you to everyone involved in resolving this ticket. I'm blown away by the rapid response and helpfulness! I'll keep my eye out for the next patch

**Comment by joshwooding** [comment]
Checkbox and Radio button have been fixed. Further QA is needed to see what other components are affected.

**Comment by adamrideout** [comment]
@joshwooding I just noticed that is alignment issue is also present in the Banner component. Would you like for me to file another issue, or are you already tracking?

**Comment by joshwooding** [comment]
> @joshwooding I just noticed that is alignment issue is also present in the Banner component. Would you like for me to file another issue, or are you already tracking?

Thanks @adamrideout, we can re-use this issue. I'll add a list to the description so we can keep track of the discovered issues.

**Comment by origami-z** [comment]
13 Nov - All items in a PR, Chromatic tested, review soon

**Comment by origami-z** [comment]
17 Nov - PR ready, waiting for other PRs to be merged first.

**Comment by origami-z** [comment]
Merged fix in #2679, pending release

**Comment by adamrideout** [comment]
Fix to radio buttons looks great! Just wanted to check, is the fix for banner component coming through as well?

**Comment by joshwooding** [comment]
Hey @adamrideout, we released [@salt-ds/core@1.12.0](https://github.com/jpmorganchase/salt-ds/releases/tag/%40salt-ds%2Fcore%401.12.0) and [@salt-ds/lab@1.0.0-alpha.22](https://github.com/jpmorganchase/salt-ds/releases/tag/%40salt-ds%2Flab%401.0.0-alpha.22) which should have fixed this across all our components as well as Banner. Please let us know if you face any issues with this version thanks.
